app/views/detailed_guide/show.html.erb

```diff
@@ -57,13 +57,13 @@
         margin_bottom: 6,
       } %>
 
-      <%= render 'govuk_publishing_components/components/govspeak', {} do %>
-        <%= raw(@content_item.govspeak_body[:content]) %>
+      <%= render "govuk_publishing_components/components/govspeak", {} do %>
+        <%= raw(content_item.body.html_safe) %>
```

Comment:
There's a bunch of lint fixing in this commit as well as the line mentioned in the description. (you could either update the description of the commit, or better yet break it into two commits - one for linting, one for the change)